src/engine/__tests__/is-authorized.test.js

```diff
@@ -0,0 +1,19 @@
+describe('is-authorized test', () => {
+  afterEach(() => {
+    jest.resetModules();
+  });
+
+  test('getAuthorization should return false', async () => {
+    const { getAuthorization } = require('../is-authorized');
```

Comment:
I didn't realize we were still exporting this from the file. We should remove it from exports and just test the two other exported functions, `isAuthorized` and `filterResponse`.